docs/en/reference/configuration.rst

```diff
@@ -131,28 +131,28 @@ Please note that if you want to use the YAML configuration option, you will need
-| transactional              | no         | true                         | Whether or not to wrap migrations in a single transaction.                       |
-|                            |            |                              |                                                                                  |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
-| migrations                 | no         | []                           | Manually specify the array of migration versions instead of finding migrations.  |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
-| check_database_platform    | no         | true                         | Whether to add a database platform check at the beginning of the generated code. |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
-| organize_migrations        | no         | ``none``                     | Whether to organize migration classes under year (``year``) or year and month (``year_and_month``) subdirectories. |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
-| connection                 | no         | null                         | The named connection to use (available only when ConnectionRegistryConnection is used). |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
-| em                         | no         | null                         | The named entity manager to use (available only when ManagerRegistryEntityManager is used). |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
++----------------------------+------------+------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
+| Name                       | Required   | Default                      | Description                                                                                                                 |
++============================+============+==============================+=============================================================================================================================+
+| migrations_paths<string, string>       | yes        | null             | The PHP namespace your migration classes are located under and the path to a directory where to look for migration classes. |
```

Comment:
Thanks for your contribution. It looks like the `Required` value isn't shown either in the screenshot / [documentation page](https://www.doctrine-project.org/projects/doctrine-migrations/en/3.8/reference/configuration.html). Can you please fix this too while you're at it?

Reply:
It should be good now